app/assets/tailwind/application.css

```
@@ -1 +1,334 @@
+  backdrop-filter: blur(12px) saturate(150%);
+  border: 1px solid rgb(255 255 255 / 0.2);
+}
+
+.dark .glass-card {
+  background-color: rgb(17 24 39 / 0.8);
+  border-color: rgb(55 65 81 / 0.2);
+}
+
+/* Premium gradient backgrounds */
+.gradient-primary {
+  background: linear-gradient(135deg, theme('colors.indigo.600') 0%, theme('colors.purple.600') 100%);
```

Biome (2.1.2)

[error] Unexpected unknown function: theme

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)